--- a/packages/ffmpeg/src/ffmpeg-default-transcoding-profile.ts
+++ b/packages/ffmpeg/src/ffmpeg-default-transcoding-profile.ts
@@ -13,13 +13,11 @@
 import { FfprobeData } from 'fluent-ffmpeg'
 
 const defaultX264VODOptionsBuilder: EncoderOptionsBuilder = (options: EncoderOptionsBuilderParams) => {
-  const { fps, inputRatio, inputBitrate, resolution } = options
-
-  const targetBitrate = getTargetBitrate({ inputBitrate, ratio: inputRatio, fps, resolution })
+  const { fps } = options
 
   return {
     outputOptions: [
-      ...getCommonOutputOptions(targetBitrate),
+      ...getVODOutputOptions(),
 
       `-r ${fps}`
     ]
@@ -192,6 +190,16 @@
   return Math.min(targetBitrate, inputBitrateWithMargin)
 }
 
+function getVODOutputOptions () {
+  return [
+    `-preset slow`,
+    `-profile:v high`,
+    `-crf 20`,
+    `-b_strategy 1`,
+    `-bf 16`
+  ]
+}
+
 function getCommonOutputOptions (targetBitrate: number, streamNum?: number) {
   return [
     `-preset veryfast`,
